server/export.js

```diff
@@ -27,9 +27,12 @@ module.exports = function(fs, db) {
             });
         }) // run the command to output the table to a file
         // replace 'annotation' with a select query to get certain rows instead of the whole thing
-        .then(db.none(`copy annotation to '/tmp/csv/export.csv' DELIMITER ',' CSV HEADER`))
+        .then(db.none(`COPY annotation TO '/tmp/csv/export.csv' DELIMITER ',' CSV HEADER`))
         .then(function() {
             // download file to client
+            if (!fs.existsSync('/tmp/csv/export.csv')){
+                fs.writeFile('/tmp/csv/export.csv', 'content')
```

Comment:
I put this catch in because otherwise the error throwing would crash the server. I agree that scale-up we don't want synchronous IO, but it should be fine for the prototype.